optimum/intel/openvino/trainer.py

```diff
@@ -346,6 +372,9 @@ def _inner_training_loop(
                 else args.max_steps * args.gradient_accumulation_steps
             )
             self.control = self.callback_handler.on_epoch_begin(args, self.state, self.control)
+            if self.compression_controller is not None:
+                self.compression_controller.scheduler.epoch_step()
+                print(self.compression_controller.statistics().to_str())
```

Comment:
Could this be done with `logger.info()` instead?

Reply:
Good catch )